public/index.html

```diff
@@ -1,43 +1,445 @@
+        }
+      }
+
+      @media (min-width: 992px) {
+        .id7-fixed-width-container,
+        .id7-main-content-area>header,
+        .id7-main-content-area>.id7-main-content {
+          width: 970px;
+        }
+
+        .id7-left-border {
+          -webkit-transform: translate(-485px,0);
+          -ms-transform: translate(-485px,0);
+          -o-transform: translate(-485px,0);
+          transform: translate(-485px,0);
+        }
+
+        .id7-right-border {
+          -webkit-transform: translate(485px,0);
+          -ms-transform: translate(485px,0);
+          -o-transform: translate(485px,0);
+          transform: translate(485px,0);
+        }
+      }
+
+      @media (min-width: 1200px) {
+        .id7-fixed-width-container,
+        .id7-main-content-area>header,
+        .id7-main-content-area>.id7-main-content {
+          width: 1170px;
+        }
+
+        .id7-left-border {
+          -webkit-transform: translate(-585px,0);
+          -ms-transform: translate(-585px,0);
+          -o-transform: translate(-585px,0);
+          transform: translate(-585px,0);
+        }
+
+        .id7-right-border {
+          -webkit-transform: translate(585px, 0);
+          -ms-transform: translate(585px, 0);
+          -o-transform: translate(585px, 0);
+          transform: translate(585px, 0);
+        }
+      }
+
+      @media (min-width: 1921px), (min-height: 1201px) {
+        .id7-left-border, .id7-right-border {
+          background-size: cover;
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <div class="id7-left-border"></div>
+    <div class="id7-fixed-width-container">
+      <header class="id7-page-header">
+        <div class="id7-utility-masthead">
+          <nav class="id7-utility-bar"></nav>
+          <div class="id7-masthead">
+            <div class="id7-masthead-contents">
+              <div class="clearfix">
+                <div class="id7-logo-column">
+                  <div class="id7-logo-row">
+                    <div class="id7-logo">
+                      <a href="http://warwick.ac.uk" title="Warwick homepage">
+                        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" alt="Warwick">
+                      </a>
+                    </div>
+                  </div>
+                </div>
+              </div>
+
+              <div class="id7-header-text clearfix">
+                <h1>
+                  <span class="id7-current-site-link">University of Warwick</span>
+                </h1>
+              </div>
+            </div>
+          </div>
+        </div>
+      </header>
+
+      <main class="id7-main-content-area" id="main">
+        <header class="id7-main-content-header">
+          <div class="id7-page-title">
+            <h1>StatusPage embed widget</h1>
+          </div>
+        </header>
+
+        <div class="id7-main-content">
+          <noscript>You need to enable JavaScript to run this app.</noscript>

```

Comment:
Do you need JS for this docs page tho?

Reply:
There's a test mode status widget on there which won't display without it